Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -334,4 +389,17 @@ class NewsTableViewController: UITableViewController, NewsCellViewModelDelegate
     func newsViewModelDidSelectReactionPicker(_ viewModel: NewsCellViewModel) {
         performSegue(withIdentifier: "reaction", sender: viewModel)
     }
+    
+    // MARK: UIViewControllerTransitioningDelegate
+    func animationController(forPresented presented: UIViewController,
+                             presenting: UIViewController,
+                             source: UIViewController) -> UIViewControllerAnimatedTransitioning? {
+        let animator = DimPresentAnimationController()
+        animator.isPresenting = true
+        return animator
+    }
+    
+    func animationController(forDismissed dismissed: UIViewController) -> UIViewControllerAnimatedTransitioning? {
+        return DimPresentAnimationController()
+    }
 }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/b3221f0a091d78ec4f94454aba4742e808f6b14d/Headlines/src/Controllers/NewsTableViewController.swift#L405"></span>File should contain 400 lines or less: currently contains 405</td>
    </tr>
  </tbody>
</table>